lib/api/src/externals/table.rs

```diff
@@ -27,7 +27,7 @@ pub struct Table {
 fn set_table_item(
     table: &dyn RuntimeTable,
     item_index: u32,
-    item: VMCallerCheckedAnyfunc,
+    item: TableReference,
```

Comment:
Yeah, I agree it's weird, it's because tables can only contain references... and the term `TableElement` was already in use by some other type. I agree that we should probably change it before shipping it, especially because this type is more public than the other one